packages/venia-concept/src/components/Checkout/__tests__/braintreeDropin.spec.js

```diff
@@ -18,26 +13,17 @@ const props = {
     onSuccess: mockSuccess
 };
 
-beforeAll(() => {
-    dropin.create = mockCreate;
-});
-
-beforeEach(() => {
-    // We clear this mock so we don't blow away the implementation.
-    mockCreate.mockClear();
```

Comment:
We've configured Jest to automatically clear mocks after every test, so you don't need to call this.